onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -1100,7 +1118,38 @@ void addObjectMethods(py::module& m, Environment& env) {
       .def_static("cuda", []() { return OrtDevice::GPU; })
       .def_static("default_memory", []() { return OrtDevice::MemType::DEFAULT; });
 
-  py::class_<OrtValue> ortvalue_binding(m, "OrtValue");
+  // See docs/C_API.md for details on what the following parameters mean and how to choose these values
+  py::class_<OrtArenaCfg> ort_arena_cfg_binding(m, "OrtArenaCfg");
+  // There is a global var: arena_extend_strategy, which means we can't use that var name here
+  ort_arena_cfg_binding.def(py::init([](size_t max_mem, int arena_extend_strategy_local,
```

Comment:
How do Python users know what values to fill in to allow ORT to choose good defaults? May be we can point them to the C API header file or add a one line note in C_API.md.

Reply:
Isn't a reference to C_API.md already present on line 1121 where the binding class is introduced ?  Should it moved down ?

Reply:
Moved the reference closer to the ctor where it is more relevant